template/YYYY-MM-threat-model-template.md

```diff
@@ -0,0 +1,90 @@
+# Title
+
+- **Author**: Your name (@YourGitHubUserName)
+
+## Overview
+
+<!--
+Provide a succinct high-level description of the system, feature, or component being analyzed. Explain why a threat model is being created for this system, feature, or component. This section should be one to three paragraphs long and understandable by someone outside the Radius team.
+-->
```

Comment:
```suggestion
<!--
Provide a succinct high-level description of the system, feature, or component being analyzed. Explain why a threat model is being created for this system, feature, or component. This section should be one to three paragraphs long and understandable by someone outside the Radius team.

Provide links to supporting documentation like feature specs and design documents rather than duplicating information.
-->
```